app/src/main/java/org/desarrolladorslp/teckersapp/model/DeliverableHeader.kt

```diff
@@ -0,0 +1,3 @@
+package org.desarrolladorslp.teckersapp.model
+
+data class DeliverableHeader(var id :String?, var title: String?,var dueDate:String?, var status:String?)
```

Comment:
Is there any reason this can't be called `Deliverable` instead of `DeliverableHeader`?

Reply:
yeah because later is going to be one call `deliverable` for the body

Reply:
for the content of the deliverable.